packages/easy-coding-standard/src/Formatter/AbstractPHPFormatter.php

```diff
@@ -74,7 +74,7 @@ function ($match) use ($noStrictTypesDeclaration): string {
 
     protected function fixContent(string $content, bool $noStrictTypesDeclaration): string
     {
-        $key = md5($content);
+        $key = bin2hex(random_bytes(16));
```

Comment:
I changed to `bin2hex(random_bytes(16));` because php file generation may cause same file when content is same. By this change, I can remove unnecessary tweak in loop per diff files.